input/pagecontent/guidance_treatment.md

```diff
@@ -13,11 +13,12 @@ If the medication is in reserve, the structured dosage instructions represent th
 
 #### Codes
 
-Implementers SHALL use the GTIN code if available. If it is not (e.g. Colchicine that was not authorized by Swissmedic but still recommended by the
+Implementers SHOULD specify a code. Use the GTIN code if available. If it is not (e.g. Colchicine that was not authorized by Swissmedic but still recommended by the
 [Schweizerische Gesellschaft für Rheumatologie](https://www.rheuma-net.ch/de/dok/sgr-dokumente/behandlung/therapie/other-therapies/519-colchicin/file)),
 then the relevant ATC code SHOULD be used.
 The ATC code may be used even if the GTIN is specified.
 Implementers MAY use other codes (e.g. pharmacode) if available.
```

Comment:
No they may not. The PMP accepts only GTIN and/or ATC as coding systems.

Reply:
ok I'll remove that line